synfig-studio/src/gui/widgets/widget_sublayer.cpp

```diff
@@ -100,24 +83,13 @@ Widget_Sublayer::set_value_desc(const synfigapp::ValueDesc &x)
 void
 Widget_Sublayer::refresh()
 {
-	typedef Gtk::TreeModel::Children type_children;
-	type_children children = enum_TreeModel->children();
-	for(type_children::iterator iter = children.begin();
-		iter != children.end(); ++iter)
-	{
-		Gtk::TreeModel::Row row = *iter;
-		if(row.get_value(enum_model.value) == value)
-		{
-			set_active(iter);
-			return;
-		}
-	}
+	set_active_id(value);
 }
 
 void
 Widget_Sublayer::set_value(std::string data)
```

Comment:
Maybe change it to `const std::string& data`?

Reply:
Done :)